.gitlab-ci.yml

```diff
@@ -21,6 +21,8 @@ stages:
   - pre_release_test
```

Comment:
Is ping sufficient to prevent issues with the docker image? Or, in other words, what does pinging the frontend server verify exactly? It verifies there is a server listening. And it doesn't verify any application behaviour expected, like loading an expected page.

IMO we need to e2e test the docker image, running the tests suite on it (see for example https://github.com/vmware/versatile-data-kit/blob/main/projects/frontend/cicd/test_e2e_data_pipelines.sh)